util-core/src/main/scala/com/twitter/util/Activity.scala

```diff
@@ -2,12 +2,13 @@ package com.twitter.util
 
 import java.util.{List => JList}
 
-import scala.collection.generic.CanBuild
 import scala.collection.JavaConverters._
 import scala.collection.mutable.Buffer
 import scala.language.higherKinds
 import scala.reflect.ClassTag
 import scala.util.control.NonFatal
+import scala.Iterable
+import scala.collection.compat._
```

Comment:
Which change requires us to enable this? What process can deduce that we need to have this import?
We are thinking about having a scalafix rule to add this import, wonder if it is doable.

Reply:
Also, will that be any compile error without this import and build against 2.12?

Reply:
@yufangong 2.12 doesn't have `Factory`, its added by compat